src/main/scala/com/typesafe/sbt/SbtGit.scala

```diff
@@ -9,7 +9,11 @@ import com.typesafe.sbt.git.ReadableGit
 import com.typesafe.sbt.git.DefaultReadableGit
 
 /** This plugin has all the basic 'git' functionality for other plugins. */
-object SbtGit extends Plugin {
+object SbtGit extends AutoPlugin {
```

Comment:
As I said, this one remains as is to try to avoid binary breakage if dependent plugins grab our keys from this object.

If we change that, we'll break the sbt-ghpages (and other dependent) plugins.